core/lib/benches/simple-routing.rs

```diff
@@ -109,7 +109,7 @@ mod benches {
 
         // Hold all of the requests we're going to make during the benchmark.
         let mut requests = vec![];
-        for route in client.rocket().routes() {
+        for route in client.manifest().routes() {
```

Comment:
I apparently can't comment on lines that haven't been modified, but note that this no longer compiles due to a missing `.await` on `Client::new()`. Can we even do async benches?

Reply:
Ah right, I keep forgetting about the benches (and the test script doesn't include them). In one sense the answer is "yes, of course" - just write `block_on` somewhere. Doing it in such a way that the benchmarks actually measure what is supposed to be measured is the hard part - as I believe we have discussed, I expect these style of benchmarks will amplify the per-request overhead of `async` and fail to demonstrate the overall advantage provided by work-stealing.

How do you feel about deferring these fixes until an upcoming PR for a blocking API for `Client`, so we can compare some approaches?

Reply:
Sounds good to me!